src/test/java/uk/gov/companieshouse/confirmationstatementapi/client/OracleQueryClientTest.java

```diff
@@ -121,6 +122,31 @@ void testGetActiveDirectorDetailsNotOkStatusResponse() {
         assertThrows(ServiceException.class, () -> oracleQueryClient.getActiveDirectorDetails(COMPANY_NUMBER));
     }
 
+    @Test
+    void testGetListActiveOfficersDetailsOkStatusResponse() throws ServiceException, ActiveOfficerNotFoundException {
```

Comment:
the throws ActiveOfficerNotFoundException can be removed as that exception is never thrown in the test